angular-ngrx-scss/src/styles.scss

```diff
@@ -1,43 +1,25 @@
 html,
 body {
```

Comment:
These attributes were adding a horizontal scrollbar to the profile page when the vertical scrollbar appeared. I think that happens because 100vw doesn't subtract the width of the scrollbar in some situations? I'll check the sign-in page and see if there's a way we can get both pages styled correctly.